octoprint_mrbeam/static/js/app/view-models/mother.js

```diff
@@ -537,6 +549,8 @@ $(function () {
                         MRBEAM_PLUGIN_VERSION +
                         ", isFirstRun: " +
                         CONFIG_FIRST_RUN +
+                        ", backendLaserCutterMode: frontend=" +
```

Comment:
since this whole section is just frontend checks and it is mentioned at the top
```suggestion
                        ", backendLaserCutterMode: " +
```

Reply:
Oh the log didn't make much sense... changed it :sweat_smile: